reference/system-databases: add description for TABLES table (#1821)
pingcap/docs: dev/reference/system-databases/information-schema.md

  [LIKE 'wild']
```

Most of the information in the table is the same as MySQL. Only two columns are newly defined by TiDB:

> * `TIDB_TABLE_ID`: to specify the internal ID of a table. This ID is unique in a TiDB cluster.

Reviewer: ```suggestion
* `TIDB_TABLE_ID`: to indicate the internal ID of a table. This ID is unique in a TiDB cluster.
```
I think 'indicate' is better.

> * `TIDDB_ROW_ID_SHARDING_INFO`: to specify the sharding type of a table. The possible values are as follows:

Reviewer: ```suggestion
* `TIDDB_ROW_ID_SHARDING_INFO`: to indicate the sharding type of a table. The possible values are as follows:
```
Ditto.

    - `"NOT_SHARDED"`: the table is not sharded.

>     - `"NOT_SHARDED(PK_IS_HANDLE)"`: the table that defines Primary Key as integer is not sharded.

Reviewer: ```suggestion
    - `"NOT_SHARDED(PK_IS_HANDLE)"`: the table that defines Primary Key as the row id is not sharded.
```

>     - `"NOT_SHARDED(PK_IS_HANDLE)"`: the table that defines Primary Key as the row id is not sharded.

Reviewer 2: ```suggestion
    - `"NOT_SHARDED(PK_IS_HANDLE)"`: the table that defines an integer Primary Key as the row id is not sharded.
```

>     - `"PK_AUTO_RANDOM_BITS={bit_number}"`: the table that defines Primary Key as integer is sharded because the table defines `AUTO_RANDOM`.

Reviewer: ```suggestion
    - `"PK_AUTO_RANDOM_BITS={bit_number}"`: the table that defines Primary Key as its row id is sharded because the Primary Key is assigned with `AUTO_RANDOM` attribute.
```

Author: @lilin90 FYI.

>     - `"PK_AUTO_RANDOM_BITS={bit_number}"`: the table that defines Primary Key as its row id is sharded because the Primary Key is assigned with `AUTO_RANDOM` attribute.

Reviewer 2: Did you confirm the meaning with @bb7133? If not, please help check.

Author: Actually I committed his suggestions for these two lines. You may refer to the conversations above.

Reviewer 2: Got it. I'll take a look.
